Sources/muterCore/MutationOperators/TokenAwareVisitor.swift

```diff
@@ -88,12 +104,36 @@ class TokenAwareVisitor: MuterVisitor {
             at: mutationRangeInCodeBlock
         )
     }
+
+    private func tryFixOperatorRange(
+        for mutatedSyntax: SyntaxProtocol,
+        in codeBlockDescription: String,
+        at nodePosition: Int
+    ) -> (start: String.Index, end: String.Index)? {
+        let mutatedSyntaxDescription = mutatedSyntax.description
+        let op = oppositeOperatorMapping[mutatedSyntaxDescription.trimmed]
+        for i in 0 ... codeBlockDescription.count {
+            let start = codeBlockDescription.index(
+                codeBlockDescription.startIndex,
+                offsetBy: nodePosition - i
+            )
+            let end = codeBlockDescription.index(
+                codeBlockDescription.startIndex,
+                offsetBy: (nodePosition + mutatedSyntaxDescription.count) - i
+            )
+
+            if String(codeBlockDescription[start ..< end]).trimmed == op {
+                return (start: start, end: end)
+            }
+        }
+
+        return nil
+    }
```

Comment:
@hoangatuan I think fixes the mutation issue, I've added a test case and the crash if not happening

```swift
if ProcessInfo.processInfo.environment["file_RelationalOperatorReplacement_1_10_9"] != nil {
  if value != "バルーンの表示判定" {

  }
} else {
  if value == "バルーンの表示判定" {

  }
}
```

Reply:
@rakaramos Can you try to run muter with this piece of code:

```swift
class A {
    public func isEnabledToShow() -> Bool {
        // バルーンの表示判定
        return activationFlag && !isDiscarded()
    }
}
```